docker-compose.yaml

```diff
@@ -1,59 +1,64 @@
 version: '3'
-
-# Other services are specified in .\lib\killrvideo-docker-common\docker-compose.yaml
 services:
-  node:
-    build: /Users/aleksandrvolochnev/killrvideo/node
+  # NodeJS Backend
+  backend:
+    build: .
+    volumes:
+      - .:/usr/src/app
     depends_on:
       - dse
     environment:
-      KILLRVIDEO_ETCD: "etcd:2379"
-      KILLRVIDEO_DSE_USERNAME: $KILLRVIDEO_DSE_USERNAME
-      KILLRVIDEO_DSE_PASSWORD: $KILLRVIDEO_DSE_PASSWORD
-      KILLRVIDEO_CASSANDRA_REPLICATION: $KILLRVIDEO_CASSANDRA_REPLICATION
-      KILLRVIDEO_LOGGING_LEVEL: $KILLRVIDEO_LOGGING_LEVEL
+      KILLRVIDEO_LOGGING_LEVEL: debug
 
-  # Start the KillrVideo web UI on port 3000
+  # Datastax Enterprise 
+  dse:
+    image: datastax/dse-server:6.0.0
+    command: [ -s -g ]
+    ports:
+      - "9042:9042"
+      - "8983:8983"
+      - "8182:8182"
+    environment:
+      DS_LICENSE: accept
+    # Allow DSE to lock memory with mlock
+    cap_add:
+      - IPC_LOCK
+    ulimits:
+      memlock: -1
+
+  # Frontend
   web:
-    image: killrvideo/killrvideo-web:1.2.6
-    #build: /Users/aleksandrvolochnev/killrvideo/web
+    image: killrvideo/killrvideo-web:3.0.0-rc1
     ports:
     - "3000:3000"
     depends_on:
     - dse
-    - etcd
     environment:
-      SERVICE_3000_NAME: web
-      KILLRVIDEO_ETCD: "etcd:2379"
-      KILLRVIDEO_DSE_USERNAME: $KILLRVIDEO_DSE_USERNAME
-      KILLRVIDEO_DSE_PASSWORD: $KILLRVIDEO_DSE_PASSWORD
-      KILLRVIDEO_CASSANDRA_REPLICATION: $KILLRVIDEO_CASSANDRA_REPLICATION
-      KILLRVIDEO_LOGGING_LEVEL: $KILLRVIDEO_LOGGING_LEVEL
+      KILLRVIDEO_LOGGING_LEVEL: debug
 
-  # The sample data generator
+  # DSE Configurator to setup DSE
+  dse-config:
+    image: killrvideo/killrvideo-dse-config:2.2.1
+    environment:
+      KILLRVIDEO_SERVICE_DISCOVERY_DISABLED: 'true'
+    depends_on:
+      - dse
+
+  # Sample Data Generator
   generator:
-    image: killrvideo/killrvideo-generator:1.2.4
+    image: killrvideo/killrvideo-generator:3.0.0-rc4
     depends_on:
       - dse
-      - etcd
+      - backend
     environment:
-      KILLRVIDEO_ETCD: "etcd:2379"
-      KILLRVIDEO_DSE_USERNAME: cassandra
-      KILLRVIDEO_DSE_PASSWORD: cassandra
-      NODE_ENV: $NODE_ENV
-      KILLRVIDEO_CASSANDRA_REPLICATION: $KILLRVIDEO_CASSANDRA_REPLICATION
-      KILLRVIDEO_LOGGING_LEVEL: $KILLRVIDEO_LOGGING_LEVEL
+      KILLRVIDEO_LOGGING_LEVEL: debug
 
-  # One instance of DataStax Studio
+  # DataStax Studio
   studio:
     image: killrvideo/killrvideo-studio:2.0.0
     ports:
-    # The Web UI exposed to our host
-    - "9091:9091"
+      - "9091:9091"
     depends_on:
-    - dse
+      - dse
     environment:
-      SERVICE_9091_NAME: studio
```

Comment:
Can we remove because of some of the new changes?

Reply:
Sorry didn't get it. Remove what?

Reply:
Oh ok, Now I see it... Github showed me different changeset, that's weird...

Reply:
Yea, I was referring to "SERVICE_9091_NAME: studio".  Sorry for not being more explicit.  I haven't done any PR's in a while and I thought github would show you the exact line I was referring to.

Reply:
Yep, this line is dead and gone :)